test.js

```diff
@@ -131,4 +133,31 @@ describe('anymatch', function() {
       matchers.pop();
     });
   });
+
+  describe('windows paths', function() {
+    anymatch.__set__('platform', 'win32');
+
```

Comment:
I just double-checked the readme, looks like I didn't document `unixify`, I'm doing that now. sorry, this would have been easier to find